components/lsp/LSP.tsx

```diff
@@ -95,45 +95,29 @@ const Testing = () => {
       contract.getCurrentTime().then((ts: ethers.BigNumber) => {
         setCurrentTime(ts.toString());
```

Comment:
Not sure how critical this call is, but currently its not provided by api. Would be nice if we could get all state related to lsp from api to avoid this contract instance.  you might be able te get current block time instead

Reply:
i changed my mind on this, i think because you need the contract instance anyway, and this might affect running in a test environment. so just leave it as is